src/main.c

```diff
@@ -1683,6 +1683,56 @@ static gboolean service_start(int argc, char **argv)
 	return TRUE;
 }
 
+static gboolean mount_start(int argc, char **argv)
+{
+	g_autofree gchar *bundlelocation = NULL;
+	g_autoptr(RaucBundle) bundle = NULL;
+	GError *error = NULL;
+	gboolean res = FALSE;
+
+	if (argc < 3) {
+		g_printerr("A file name must be provided\n");
+		r_exit_status = 1;
+		return FALSE;
+	}
+
+	if (argc > 3) {
+		g_printerr("Excess argument: %s\n", argv[3]);
+		r_exit_status = 1;
+		goto out;
+	}
+
+	bundlelocation = resolve_bundle_path(argv[2]);
+	if (bundlelocation == NULL)
+		goto out;
+	g_debug("input bundle: %s", bundlelocation);
+
+	res = check_bundle(bundlelocation, &bundle, TRUE, &error);
+	if (!res) {
+		g_printerr("%s\n", error->message);
+		g_clear_error(&error);
+		goto out;
+	}
+
+	g_message("bundle payload size: %"G_GOFFSET_FORMAT, bundle->size);
+
+	res = mount_bundle(bundle, &error);
+	if (!res) {
+		g_printerr("%s\n", error->message);
+		g_clear_error(&error);
+		goto out;
+	}
+
+	g_message("bundle mounted at: %s", bundle->mount_point);
+
+	/* The mount keeps the block device active. */
+	exit(0);
+
+out:
+	r_exit_status = res ? 0 : 1;
+	return TRUE;
```

Comment:
Why using `exit` and setting `r_exit_status` with a ternary. `out` label can it not exist that in case of error right ?

Reply:
I've made this consistent. `r_exit_status` is now set to `1` in `out:` and we return `FALSE`.